CMakeLists.txt

```diff
@@ -696,6 +696,7 @@ include (cmake/package.cmake)
 # so rpm.cmake depends on package.cmake.
 #
 include (cmake/rpm.cmake)
+include (cmake/SetFiberStackSize.cmake)
```

Comment:
Nit: Judging by the comment above, it looks like this statement is a part of the "RPM build environment". It's probably better to move it here to avoid confusion:

https://github.com/tarantool/tarantool/blob/294f2d616c12334b0b9344ee21399bea932c3538/CMakeLists.txt#L73-L75

Reply:
It was hard to find the right place in this CMakeLists.txt, so I've placed fiber stack size tweaks nearby other flags setup. TBH, you found better place for it, so I moved `include` there.

```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index d7a5a4163..bff6e559d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -72,6 +72,7 @@ include(cmake/module.cmake)
 include(cmake/thread.cmake)
 include(cmake/hardening.cmake)
 include(cmake/prefix.cmake)
+include(cmake/SetFiberStackSize.cmake)
 
 add_compile_flags("C;CXX" ${HARDENING_FLAGS})
 set(DEPENDENCY_CFLAGS "${DEPENDENCY_CFLAGS} ${HARDENING_FLAGS}")
@@ -696,7 +697,6 @@ include (cmake/package.cmake)
 # so rpm.cmake depends on package.cmake.
 #
 include (cmake/rpm.cmake)
-include (cmake/SetFiberStackSize.cmake)
 
 add_subdirectory(src)
 add_subdirectory(extra)
```